src/Containers/Microsoft.NET.Build.Containers/Registry.cs

```diff
@@ -526,9 +727,15 @@ private HttpClient GetClient()
 
     private HttpClient CreateClient()
     {
-        HttpMessageHandler clientHandler = new AuthHandshakeMessageHandler(new SocketsHttpHandler() { PooledConnectionLifetime = TimeSpan.FromMilliseconds(10 /* total guess */) });
-
-        if(IsAmazonECRRegistry)
+        HttpMessageHandler clientHandler = new AuthHandshakeMessageHandler(new SocketsHttpHandler()
+        {
+            PooledConnectionLifetime = TimeSpan.FromMilliseconds(10 /* total guess */),
+            // disabling cookies prevents CSRF tokens from being sent - some servers send these and
```

Comment:
This is a fix for Harbor registry specifically (though it makes sense for all registries). Other golang based clients disable cookies so that UI-only cookies like CSRF tokens don't impact communication, so we should as well.